test/autorest/arraygroup/zz_generated_array.go

```diff
@@ -24,7 +24,8 @@ type ArrayOperations interface {
 	GetArrayNull(ctx context.Context, options *ArrayGetArrayNullOptions) (*StringArrayArrayResponse, error)
 	// GetArrayValid - Get an array of array of strings [['1', '2', '3'], ['4', '5', '6'], ['7', '8', '9']]
 	GetArrayValid(ctx context.Context, options *ArrayGetArrayValidOptions) (*StringArrayArrayResponse, error)
-	// GetBase64URL - Get array value ['a string that gets encoded with base64url', 'test string' 'Lorem ipsum'] with the items base64url encoded
+	// GetBase64URL - Get array value ['a string that gets encoded with base64url', 'test string' 'Lorem ipsum'] with the items
```

Comment:
I think this truncation happened due to the use of `comment()`.  I now recall that in track 1 we had an issue with `comment()` being somewhat non-deterministic and would sometimes change where the line-break would occur.  So you'd get a diff in the codegen even if the input didn't change.  I think we can fix this via the `factor` and/or `maxLength` params.